### PR TITLE
Fixes Kerberos/Delta Research Directors monitor shows Detectives Office camera

### LINKED ISSUE #24304: Kerberos/Delta Research Directors monitor shows Detectives Office camera
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1630

### Issue Description

The research director monitor in their office on Kerberos/Delta shows Detectives office camera as an option along with all the other normal science cameras.

### What did you expect to happen?

Not to be able to spy on the detective sleeping on the job.

### What happened instead?

I saw them... I saw it all.

### Why is this bad/What are the consequences?

This is not the case on other stations, so would be weird for this one to be the exception. Also RD doesn't need to supervise the detective, right?

### Steps to reproduce the issue.

1. Spawn in as RD on NSS Kerberos
2. Access the Research Director Monitor in their office
3. Spy on the detective instead of your own subordinates

### When did the problem start happening?

¯\_(ツ)_/¯

### Extra information

_No response_

### Relevant log output/runtime error

_No response_

### PR DESCRIPTION
**Pull request by Fluffyzack21**
## What Does This PR Do
 Fixes the camera in the detective office being labeled wrong, on meta and on delta

Fixes #24304 

## Why It's Good For The Game
Not to be able to spy on the detective sleeping on the job.

## Images of changes

Screenshot 2024-02-24 204139
Screenshot 2024-02-24 204015

## Testing
Spawned as the RD, and not to be able to spy on the detective sleeping on the job
 
## Changelog
:cl:
tweak: Fixed the label for the security camera iso RD can't spy on the detective in the detective office on delta and meta station
fix: moved a light and a light switch not be over lapping on other things on meta and delta station
/:cl:

### CODE REVIEW

**Comment by Henri215** [comment]
image

Specify that its on detective office.

Update the PR description to include information about the changes on Meta, not only on changelog

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -60089,17 +60089,12 @@
...
-/obj/machinery/light_switch{
-	dir = 8;
-	name = "east bump";
-	pixel_x = 24
-	},
+/obj/item/kirbyplants/plant24,
+/obj/machinery/atmospherics/unary/vent_scrubber/on,
 /obj/machinery/camera{
-	c_tag = "Detective's Office";
+	c_tag = "Brig Detective's Office";
```
Comment (Henri215) [inline _maps/map_files/MetaStation/MetaStation.dmm:60095]: brig detective office?

Comment (Fluffyzack21) [reply _maps/map_files/MetaStation/MetaStation.dmm:60095]: That is how box station c_tag named it as a camera in detective office

Comment (Fluffyzack21) [reply _maps/map_files/MetaStation/MetaStation.dmm:60095]: okay only farragus the right C_tag, but the other stations do not have

**Review by Henri215 — APPROVED** [review]

**Pull request merged**